src/components/Dropdown/dropdownHelper.ts

```diff
@@ -25,22 +26,30 @@ export function getContentPositionStyle(
     left: number
   },
 ) {
-  const position: Position = {
+  const contentBox: ContentBoxStyle = {
     top: 'auto',
     left: 'auto',
+    maxHeight: '',
   }
+  const triggerAlignCenter = triggerRect.left + (triggerRect.right - triggerRect.left) / 2
 
   if (triggerRect.bottom + contentSize.height <= windowSize.height) {
-    position.top = `${scroll.top + triggerRect.bottom}px`
+    contentBox.top = `${scroll.top + triggerRect.bottom}px`
+  } else if (triggerRect.top - contentSize.height >= 0) {
+    contentBox.top = `${scroll.top + triggerRect.top - contentSize.height}px`
   } else {
-    position.top = `${scroll.top + triggerRect.top - contentSize.height}px`
+    const paddingBottom = 10
+    contentBox.top = `${scroll.top + triggerRect.bottom}px`
+    contentBox.maxHeight = `${windowSize.height - triggerRect.bottom - paddingBottom}px`
```

Comment:
I'm worry about case of the window bottom space is not enough.
This is un extreme example, but in case the below, the user hard to reads context in content.

![image](https://user-images.githubusercontent.com/36581513/67921243-7677ea00-fbea-11e9-9066-066d05800ef5.png)

So I think good solution is to determine the display position of the content based on whether the button is located above or below the window.